types/node/v16/http.d.ts

```diff
@@ -155,6 +155,22 @@ declare module 'http' {
          * @default false
          */
         insecureHTTPParser?: boolean | undefined;
+        /**
+         * If set to `true`, it disables the use of Nagle's algorithm immediately after a new incoming connection is received.
+         * @default false
+         */
+        noDelay?: boolean | undefined;
+        /**
+         * If set to `true`, it enables keep-alive functionality on the socket immediately after a new incoming connection is received,
+         * similarly on what is done in `socket.setKeepAlive([enable][, initialDelay])`.
+         * @default false
+         */
+        keepAlive?: boolean | undefined;
+        /**
+         * If set to a positive number, it sets the initial delay before the first keepalive probe is sent on an idle socket.
+         * @default 0
+         */
+        keepAliveInitialDelay?: number | undefined;
```

Comment:
I think those were introduced quite recently, only in 16.15.0. Can we have `@since` JSDoc tag added, please?
https://nodejs.org/docs/latest-v16.x/api/http.html#httpcreateserveroptions-requestlistener

Reply:
Added.